recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dump_fw_log.sh

```diff
@@ -0,0 +1,5 @@
+while true; do
+      echo "----------------------"
+#      cat /sys/kernel/debug/pvr/firmware_trace | grep "OSid = 1";
```

Comment:
An odd line?

Reply:
not really, commented one greps for OSID 1 specifically